pages/coupon/list.js

```diff
@@ -0,0 +1,72 @@
+import CommerceLayout from '@components/common/CommerceLayout';
+import SiteHead from '@components/common/SiteHead.js';
+import styles from '@styles/Home.module.scss';
+import UserCouponList from '@components/coupon/UserCouponList';
+import CouponList from '@components/coupon/CouponList';
+import { GET_CUSTOM } from '@apis/defaultApi';
+import { useEffect, useState } from 'react';
+
+export default function ShowCouponList() {
+  const [role, setRole] = useState('');
+  const [userId, setUserId] = useState(1);
+  const [sellerId, setSellerId] = useState();
+
+  // TODO:
+  // setUserId(현재로그인되어있는userId-세션,쿠키 등에서 얻어올 것임);
+
+  useEffect(() => {
+    GET_CUSTOM(`/user/role`, { userId }).then((res) => {
+      console.log(res);
+      console.log(res.data);
+
+      if (res.message === 'ROLE_USER') {
+        console.log('userId is ' + res.data);
+        setRole('USER');
+        // setUserId(res.data);
+      }
+      if (res.message === 'ROLE_SELLER') {
+        console.log('sellerid is ' + res.data);
+        setRole('SELLER');
+        setSellerId(res.data);
```

Comment:
테스트코드랑 사용하지 않는 코드 지워주세요
```suggestion
      if (res.message === 'ROLE_USER') {
        setRole('USER');
      }
      if (res.message === 'ROLE_SELLER') {
        setRole('SELLER');
        setSellerId(res.data);
```

Reply:
반영완료~